pkg/config/dynamic/http_config.go

```diff
@@ -48,6 +48,7 @@ type Router struct {
 	Rule        string           `json:"rule,omitempty" toml:"rule,omitempty" yaml:"rule,omitempty"`
 	Priority    int              `json:"priority,omitempty" toml:"priority,omitempty,omitzero" yaml:"priority,omitempty" export:"true"`
 	TLS         *RouterTLSConfig `json:"tls,omitempty" toml:"tls,omitempty" yaml:"tls,omitempty" label:"allowEmpty" file:"allowEmpty" kv:"allowEmpty" export:"true"`
+	LoopStop    bool             `json:"-" toml:"-" yaml:"-" label:"-" file:"-"`
```

Comment:
This option is not useful for now and this option can be introduced in another form (like a int instead of bool)
This option should be removed.

Reply:
Why this option is not useful? I don't have the full context. 

This option is private, no one can configure it. I don't see the benefit of having an `int`.

You suggest removing this option but you also suggest using an `int` instead of a  bool. I'm a bit lost to be honest with what you are looking for.

Reply:
My fault I read it too quickly.